opennms-doc/guide-admin/src/asciidoc/text/telemetryd/protocols/netflow5/interface.adoc

```diff
@@ -163,6 +163,19 @@ Credentials for each node must be stored in `${OPENNMS_HOME}/etc/elastic-credent
 </elastic-credentials>
 ```
 
+===== Configure Deep Dive Tool  URL
+
+In order to directly access flows from OpenNMS Web UI, deep dive tool url needs to be configured in opennms.properties.
```

Comment:
`{opennms-product-name}` should be used instead of `OpenNMS` to be consistent with the rest of the documentation.